Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/v1beta3/defaults.go

```diff
@@ -231,4 +233,58 @@ func defaultSecurityContextConstraints(scc *SecurityContextConstraints) {
 	if len(scc.SupplementalGroups.Type) == 0 {
 		scc.SupplementalGroups.Type = SupplementalGroupsStrategyRunAsAny
 	}
+
+	// defaults the volume slice of the SCC.
+	// In order to support old clients the boolean fields will always take precedence.
+	defaultAllowedVolumes := fsTypeToStringSet(scc.Volumes)
+
+	// assume a nil volume slice is allowing everything for backwards compatibility
```

Comment:
I think this should all be a custom conversion... that keeps us from needing a new field, new deep copy, etc